evdns.c

```diff
@@ -863,6 +863,19 @@ reply_schedule_callback(struct request *const req, u32 ttl, u32 err, struct repl
 		&d->deferred);
 }
 
+
+#define _QR_MASK    0x8000
```

Comment:
maybe define them as `unsigned int`, i.e. `0x8000U` what do you think?

Reply:
done.